KeyVault.Acmebot/KeyVault.Acmebot.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.4.3" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.13" />
     <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.11" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Json.NET is ready to use, so do not install another version.

Reply:
Package reference removed.